paper/paper.md

```diff
@@ -11,36 +11,57 @@ authors:
     orcid: 0000-0002-0589-5187
     affiliation: 1
   - name: Bryan Palmintier
+    orcid: 0000-0002-1452-0715
     affiliation: 1
   - name: Aadil Latif
     affiliation: 1
   - name: Kwami Senam Sedzro
+    orcid: 0000-0002-2107-8662
     affiliation: 1
   - name: Sherin Ann Abraham
+    orcid: 0000-0002-4214-3267
     affiliation: 1
   
 affiliations:
  - name: National Renewable Energy Laboratory (NREL), Golden, CO, USA
    index: 1
-date: 1 August 2023
+date: 1 February 2024
 bibliography: paper.bib
 ---
 
 # Summary
 
-Understanding the impact of disaster events on people's ability to access critical service is key to designing appropriate programs to minimize the overall impact. Flooded roads, downed power lines, flooded power substation etc. could impact access to critical servies like electricity, food, healthcare and more. The field of disaster modeling is still evolving and so is our understanding of how these events would impact our critical infrastrctures such power grid, hospitals, groceries, banks etc.
+Understanding the impact of extreme events on people's ability to access critical service is key to designing appropriate programs to minimize the overall impact. In the event of a disaster, damage to the electric system and related infrastructure (e.g. downed power lines, flooded equipment, hacked communication systems, damaged roads,  etc.) can impact people's access to critical services including not just electricity but also shelter, food, healthcare, and more.  Moreover, the individual and community ease of service access--both before and during such events--may be correlated with household income and other demographics, with some historically disadvantaged communities potentially experiencing more difficult access to critical services. There is a key need to better understanding such impacts and evaluate options to improve resilience and equity. The Equity and Resilience Analysis for electric Distribution systems (ERAD) tool is a free and open-source software package designed to be used by researchers, utilities, local governments, community groups, and other stakeholders to fill this need.
 
-ERAD is a free, open-source Python toolkit for computing equity and resilience measures in the face of hazards like earthquakes and flooding. It uses graph database to store data and perform computation at the household level for a variety of critical services that are connected by power distribution network. It uses asset fragility curves, which are functions that relate hazard severity to survival probability for power system assets including cables, transformers, substations, roof-mounted solar panels, etc. recommended in top literature. Programs like undergrounding, microgrid, and electricity backup units for critical infrastructures may all be evaluated using metrics and compared across different neighborhoods to assess their effects on equity and resilience.
+# Statement of need
```

Comment:
Per the JOSS website, the Statement of Need "places [the tool] in the context of related work" (see [https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain]. This could provide a bit more depth to the motivation if needed, but the key thing--which was previously missing--is how it relates to other research work and tools in the area. Effectively a lit review. I've taken a stab at starting this, but there are sill some gaps to fill